### PR TITLE
updated repository urls

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

**Comment by brendarearden** [comment]
https://helm.sh/blog/new-location-stable-incubator-charts/ << this pr is related to these changes here

File: __fixtures__/with-subchart/requirements.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 dependencies:
   - name: postgresql
-    repository: https://kubernetes-charts.storage.googleapis.com
+    repository: https://charts.helm.sh/stable/
```
Comment (brendarearden) [inline __fixtures__/with-subchart/requirements.yaml:3]: looks good! not sure if we need the trailing slash or not.

**Review by brendarearden — APPROVED** [review]
LGTM

**Comment by brendarearden** [comment]
Do we need a version bump, or are we going to overwrite the tag / build?

**Review by superseb — APPROVED** [review]
CI just does a build so if the trailing slash is tested, all good

**Comment by paynejacob** [comment]
@brendarearden I think this would just be the rancher2 version

**Comment by brendarearden** [comment]
@paynejacob I think rancher2 is correct, since the change isn't that big. 

Then a corresponding pr would need to be update here: https://github.com/rancher/rancher/blob/master/Dockerfile.dapper#L12 - and you will need to do a master, 2.5 and 2.4 PR - I would also ask denise if we need to port it to any other versions

**Pull request merged**